dotnet/samples/GettingStarted/AgentWithPurview/Program.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// This sample shows how to create and use a simple AI agent with Azure OpenAI Responses as the backend.
```

Comment:
I'm not sure there should be a Purview sample in the "Getting Started" section, since ostensibly the samples in there should target core/basic scenarios involved in building agents (admittedly there's probably other stuff in there that can be cleaned up).